playbooks/roles/os_temps/tasks/login_to_cluster.yml

```diff
@@ -1,4 +1,4 @@
 ---
 # login to the cluster
 - name: "Login to the Openshift cluster"
-  shell: "{{ oc_bin }} login {{cluster_ip.stdout }} --username={{ username }} --password={{ password }} --insecure-skip-tls-verify=true"
\ No newline at end of file
+  shell: "{{ oc_bin }} login {{ cluster_ip }} --username={{ username }} --password={{ password }} --insecure-skip-tls-verify=true"
```

Comment:
we don't need .stdout anymore?

Reply:
`cluster_ip` is now a fact. Either we take ` cluster_ip_register.stdout` or` openshift_cluster_ip`

Reply:
cluster_ip makes sense I didn't connect that you had set it as a fact you can resolve this question.